libs/sdk-backend-mockingbird/src/recordedBackend/dashboards.ts

```diff
@@ -67,23 +107,31 @@ export class RecordedDashboards implements IWorkspaceDashboardsService {
             return Promise.reject(new UnexpectedResponseError("Not Found", 404, {}));
         }
 
-        return Promise.resolve(recording.obj.dashboard);
-    }
+        if (isDashboardRecording(recording)) {
+            return Promise.resolve(recording.obj.dashboard);
+        }
+
+        return Promise.resolve(recording);
```

Comment:
This should probably return `localDashboards` too?

Reply:
it does.. the findDashboardRecording looks up into the localDashboard; i have renamed the findDashboardRecording so that it better fits the new responsibilities

Reply:
I'm blind, sorry.